compose/container.py

```diff
@@ -63,6 +63,11 @@ def number(self):
             return None
 
     @property
+    def ip(self):
+        self.inspect_if_not_inspected()
```

Comment:
You don't need to call `self.inspect_if_not_inspected()` explicitly anymore, it's called by `self.get(...)`